### PR TITLE
Support Notification-style Publish.

### PR DESCRIPTION
**Pull request by evshary**
As @PLeVasseur mentioned in #6, support Notification-style Publish.

### CODE REVIEW

File: tests/integration_test.rs [hunk 1]
```diff
@@ -26,24 +26,45 @@ use up_rust::{
 use uprotocol_zenoh::UPClientZenoh;
 use zenoh::config::Config;
 
-fn create_utransport_uuri() -> UUri {
-    UUri {
-        entity: Some(UEntity {
-            name: "body.access".to_string(),
-            version_major: Some(1),
-            id: Some(1234),
+fn create_utransport_uuri(index: u8) -> UUri {
```
Comment (PLeVasseur) [inline tests/integration_test.rs:29]: Is the idea that you'd in the future keep adding more `UUri`s in here for testing purposes and you want to have a single factory function for creating them?

File: src/utransport.rs [hunk 2]
```diff
@@ -482,7 +482,13 @@ impl UTransport for UPClientZenoh {
                         )
                     })?;
                 // Get Zenoh key
-                let topic = attributes.clone().source;
+                let topic = if attributes.sink.is_some() {
```
Comment (PLeVasseur) [inline src/utransport.rs:485]: Yup, seems like this is what we need :slightly_smiling_face: 

Just wanna get some input from @sophokles73

Comment (sophokles73) [reply src/utransport.rs:485]: It is correct to assume that you are dealing with a *Notification* if *sink* is not empty and a *Publish* otherwise.

**Review by PLeVasseur — APPROVED** [review]
Aside from the nit comment on the unit tests, I'll go ahead and approve :slightly_smiling_face:

File: tests/integration_test.rs (hunk 1 above)
Comment (evshary) [reply tests/integration_test.rs:29]: You're right. That's my intention.
Feel free to give me your opinions about this. I'm still wondering whether there is a more elegant way to do so.

**Pull request merged**